docs/development/debugging.md

```diff
@@ -0,0 +1,65 @@
+(debugging)=
+# Interactive Debugging
+
+[Chromium has support for DWARF
+info](https://developer.chrome.com/blog/webassembly/) which can be very helpful
+for debugging in certain circumstances. To build Pyodide with DWARF, you should
+set `EXTRA_CFLAGS="-g4"` and `EXTRA_LD_FLAGS="-g4 --source-map-base
```

Comment:
We'll have to update this in the future with https://github.com/emscripten-core/emscripten/blob/main/ChangeLog.md#2017-04102021